src/components/Form/TextInput.tsx

```diff
@@ -130,8 +126,8 @@ class BaseTextInput extends Component<TextInputProps> {
               {...passProps}
               className={classnames(inputBaseClassName, inputClassName)}
               type={type}
-              {...getDescribedBy()}
```

Comment:
this change works as it should, though the jury is still out on which approach is more appropriate.

Reply:
I'd argue that there might not be one best approach for this that fits all cases. Some things can be wrapped inside a label while other can't or may not work as desired. Might be better to use similar approach for everything since screen readers and browsers do not the same way => many different approaches with currently similar end result are not guaranteed to work the same way in the future.

Reply:
Luckily these are things we can adjust later if need be without breaking the API. I like the 'inside the label' approach since the read order is the most logical in that one at the moment.